src/Native/Unix/configure.cmake

```diff
@@ -362,6 +362,34 @@ check_prototype_definition(
 
 check_cxx_source_compiles(
     "
+    #include <stdlib.h>
+
+    int main()
+    {
+        char* path = strdup(\"abc\");
+        int ret = mkstemps(path, 3);
```

Comment:
alas, the compiler errors out on that, too: `error: null passed to a callee which requires a non-null argument [-Werror,-Wnonnull]`